ts-apps/decision-logic-visualizer/src/lib/displayers/flow/sf-custom-nodes/ubool-var.svelte

```diff
@@ -17,6 +17,9 @@ https://github.com/xyflow/xyflow/blob/migrate/svelte5/packages/svelte/src/lib/co
 <!-- Need to use data.bleh to maintain reactivity -- can't, e.g., do `const bleh = data.bleh` 
 TODO: Look into why this is the case --- are they not re-mounting the ubool-var component? 
 -->
+<!-- accessibility: Using aria-label instead of Tailwind’s sr-only because 
+ aria-label gives screen readers the intended action without also reading the “+” symbol 
+ 'Expand nested clause' is a concise, user-friendly hint tailored for legal drafters. --> 
```

Comment:
No need to have `'Expand nested clause' is a concise, user-friendly hint tailored for legal drafters.` in the comment